Sources/AblyAssetTrackingCore/Model/Location.swift

```diff
@@ -0,0 +1,100 @@
+
+import Foundation
+
+public struct Location: Equatable {
+    
+    public let coordinate: LocationCoordinate
```

Comment:
Every other field has a comment but this one doesn't, please add a comment for this field :wink:

Reply:
> Every other field has a comment but this one doesn't, please add a comment for this field 😉

added here e6d96ed5ae10cb04c958b27897912a546674995a